electrical/calibration.MD

```diff
@@ -0,0 +1,183 @@
+			*  (Robo- Claw 1 is address 128, 2 is address 129, etc)
+		* Baudrate: 115200
+			
+	(c) Battery:
+			
+		* Max Main Battery: 17.5V 
+		* Min Main Battery: 11.5V
+		* Max Logic Battery: 5.5V Min Logic Battery: 4.5V
+
+	(d) Motors:
+
+		* We will be updating the motor current parameters after calibration is complete. 
+
+	(e) I/O:
+
+		* Encoder 1 Mode: Quadrature for addresses 128,129,130; Absolute for addresses 131 and 132
+		* Encoder 2 Mode: Quadrature for addresses 128,129,130; Absolute for addresses 131 and 132
+
+After changing these settings, make sure to save them! (Device − > Save Settings )
+
+
+
+
+## 1.2 Drive Motor Calibration
+
+7. Perform the following steps for the RoboClaw addresses 128,129, and 130:
+
+	* Go to the PWM settings tab
+
+
+![](../images/calibration/is4.png)
+ **Figure 4**
+  
+	(a) Under the control pane, slowly move the slider bar up for Motor1.
+
+![](../images/calibration/is5.png)
+**Figure 5**
+
+	(b) Verify that when the forward signal is sent to the motor (the Motor1 slider is above 0), the wheel spins in the direction that would move the rover as a whole forward (note this is different clockwise vs counterclockwise based on which wheel you are testing). If the wheel moves backwards with respect to the rover direction, then under the General Settings tab click the checkbox labeled ’Reverse M Relative Direction based on which motor is wrong.
+
+	(c) Now as you drive M1 motor forward (which now corresponds to the rover moving forward), verify that M1 Encoder value increases1. If the encoder value de- creases go to the General tab and under I/O click the Invert checkbox for the Encoder Mode
+ 
+	(d) Repeat steps 7b and 7c for M2 motor.
+	
+8. Once both motors are spinning the correct direction and the encoders respond accordingly when commanded through the PWM Signal tab, move on to the Velocity Settings tab.
+
+![](../images/calibration/is6.png)
+**Figure 6**
+
+Here we’ll be tuning the motor settings. We need to do this in order to have fluid control of the motor, and it needs to be done for each individual motor as they will all differ slightly out of the box. Therefore, each motor must be be individually tuned. For more information on motor PID controls and what they do you can visit:
+
+* [An Introduction to PID Control with DC Motor](https://medium.com/pollenrobotics/an-introduction-to-pid-control-with-dc-motor-1fa3b26ec661)
+* [The Working of a PID Controller](https://www.elprocus.com/the-working-of-a-pid-controller/)
+
+There has been significant feedback that the BasicMicro Motion Studio auto tune sometimes does not work, and so we have included a section at the end of this document about how to tune the motors yourself. 
+
+	(a) Begin by entering in the QPPS (quadrature pulses per second), or the maximum number of encoder ticks per second. To find this value, drive the motor at max speed in the PWM setting and look at the box labeled ’M1/M2 Speed’.
```

Comment:
because of github formatting, this line ends up wrapped in a scroll box. I would use #### N.N.N.a